src/modules/aws_ecr/index.ts

```diff
@@ -31,16 +31,18 @@ class RepositoryImageMapper extends MapperBase<RepositoryImage> {
     return Object.is(a.registryId, b.registryId);
   };
 
-  async repositoryImageMapper(image: ImageAws, ctx: Context, type: string, region?: string) {
+  async repositoryImageMapper(image: ImageDetail, ctx: Context, type: string, region?: string) {
     const out = new RepositoryImage();
 
     // id is generated with imageDigest + tag
-    if (!image.imageId?.imageDigest) {
+    if (!image.imageDigest) {
       throw new Error('Invalid repository image');
     }
-    out.imageDigest = image.imageId.imageDigest;
-    out.imageTag = image.imageId.imageTag ?? '<untagged>';
-
+    const imageTag = image.imageTags?.[0] ?? '<untagged>';
+    out.imageDigest = image.imageDigest;
+    out.imageTag = imageTag;
+    if (image.imageSizeInBytes) out.sizeInMB = Math.round(image.imageSizeInBytes / 1024 / 1024);
```

Comment:
Maybe we should make sure `image.imageSizeInBytes` is a number? I think this would do the work
```suggestion
    if (image.imageSizeInBytes) out.sizeInMB = Math.round(+image.imageSizeInBytes / 1024 / 1024);
```

Reply:
it is typed into a number